tests/message/test_messages.py

```diff
@@ -60,6 +61,7 @@ def test_message_want_to_compute_task(self):
         concent_enabled = random.random() > 0.5
         provider_public_key = provider_ethereum_public_key = (
             encode_hex(cryptography.ECCx(None).raw_pubkey))
+        task_header = TaskHeaderFactory()
         msg = message.WantToComputeTask(
```

Comment:
This whole method should be removed, and `tests/messages/test_tasks.py::WantToComputeTaskTest` should use `SerializationMixin`. But it's a little bit on the edge of this PR (not mandatory)

Reply:
will do ;)